subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/GroovyDslMultiProjectJvmApplicationInitIntegrationTest.groovy

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.buildinit.plugins
+
+
+import org.gradle.buildinit.plugins.internal.modifiers.BuildInitDsl
+
+class GroovyDslMultiProjectJvmApplicationInitIntegrationTest extends AbstractMultiProjectJvmApplicationInitIntegrationTest {
```

Comment:
If you want, you can put those two basically empty classes in the same file as `AbstractMultiProjectJvmApplicationInitIntegrationTest`. Though you don't have to.

Reply:
Right, Groovy with multiple top level public classes.  Will do.